app/articles/server/api/api.js

```diff
@@ -0,0 +1,104 @@
+import { Restivus } from 'meteor/nimble:restivus';
+import _ from 'underscore';
+
+import { processWebhookMessage } from '../../../lib';
+import { API } from '../../../api';
+import { settings } from '../../../settings';
+import * as Models from '../../../models';
+
+const Api = new Restivus({
```

Comment:
Note: Revisit the security

Reply:
Apart from token checking, what else I have to check?